Description: gzip in wheezy lacks --keep option, so avoid using it.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/lib/Sbuild/ResolverBase.pm
+++ b/lib/Sbuild/ResolverBase.pm
@@ -1445,8 +1445,8 @@
     closedir($dh);
 }
 
-system('gzip', '--keep', '--force', 'Packages') == 0 or die "gzip failed: $?\n";
-system('gzip', '--keep', '--force', 'Sources') == 0 or die "gzip failed: $?\n";
+system('gzip -c --force Packages > Packages.gz') == 0 or die "gzip failed: $?\n";
+system('gzip -c --force Sources  > Sources.gz' ) == 0 or die "gzip failed: $?\n";
 
 my $packages_md5 = hash_file('Packages', 'md5sum');
 my $sources_md5 = hash_file('Sources', 'md5sum');
